lighthouse-core/test/config/index.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const ConfigParser = require('../../config');
+const defaultConfig = require('../../config/default.json');
+const IsOnHTTPS = require('../../audits/is-on-https');
+const assert = require('assert');
+const path = require('path');
+
+/* global describe, it*/
+
+describe('ConfigParser', () => {
+  it('returns default config', () => {
+    const config = ConfigParser.parse(undefined);
+    assert.deepEqual(config, defaultConfig);
```

Comment:
this is one of the weird things about altering the config object in place. This *is* testing that the `defaultConfig` was used and returned, but only because we're modifying the `default.json` module itself.

Since they're just references to the same object, `assert.strictEqual(config, defaultConfig)` might make that fact a little more clear.